.gitlab-ci.yml

```diff
@@ -5,10 +5,33 @@ before_script:
   - pip install awscli --upgrade
   - test $SPEL_CI == "true" && echo "export SPEL_VERSION=$(date +%Y.%m.dev%s)" >> .variables || true
 
-build_job:
+commercial_build_job:
+  variables:
+    AWS_REGION: "us-east-1"
+    SOURCE_AMI_CENTOS7_HVM: "ami-090b9dabe1c9f40b3"
+    SOURCE_AMI_RHEL7_HVM: "ami-0394fe9914b475c53"
   script:
-      - source .variables
-      - make
+    - source .variables
+    - export AWS_REGION=$AWS_REGION
```

Comment:
I _think_ these exports are superfluous... The `variables` section should export them into the env already...

Reply:
Fixed.